src/Control/Effect/Choose.hs

```diff
@@ -52,7 +52,7 @@ import qualified Data.Semigroup as S
 --
 -- @since 1.0.0.0
 (<|>) :: Has Choose sig m => m a -> m a -> m a
-(<|>) a b = send (Choose (bool b a))
+a <|> b = send (Choose (bool b a))
```

Comment:
I really couldn’t tell you why I wrote it like that but when I noticed it I had to fix it.